app/views/projects/projects/show.html.erb

```diff
@@ -1,21 +1,85 @@
-<h1><%= @project.title %></h1>
+<div class="row">
+<nav class="projects-nav">
```

Comment:
Probably makes sense to extract this out to a partial since we know we'll be using it across project pages.

Reply:
Done